### ISSUE TITLE
Interfaces created through autoload_server cannot affect the document

### ISSUE THREAD

**Issue by lukebarnard1** [bokeh/bokeh#2118]
I've made an interface which has multiple data sources, all affecting each other, and it works within an iFrame or by browsing to the bokeh-server pages. So the document is stored there and when I interact from either of these, changes are made and the document changes are pushed back to the client.

However, using autoload_server and embedding the html for some reason means that only store_objects can be used to push changes to the document (as with the animated.py example) but calling session.poll_document only listens for changes coming from the bokeh-server document page. Alternatively, the autoload_server client javascript is just not pushing it's changes to the server, but this seems unlikely as the server is receiving a PATCH and PUT from both.

In summary, the client with html generated by autoload_server has no interactivity whereas the client that browses to the bokeh pages does have interactivity. Also, the changes made on the bokeh page does update the autoload_server html page.

**Comment by lukebarnard1** [comment]
I have found a work-around using my own code to pass a docid and objid using a Django template. These go into this JavaScript:
```python
container = $("#PlotPane");
Bokeh.set_log_level("info")
Bokeh.embed.add_plot_server(container, "{{dashboard.bokeh_docid}}", "{{dashboard.bokeh_objid}}", false);
```
This is then run when the the page has loaded in the browser (from within a callback passed to $(document).ready()) and a view is generated akin to the one that can be seen by browsing to the bokeh-server itself. Interaction works and no iFrames are required.

Also, the docid and objid are acquired from the session object:

```python
def __init__(self):
         self.session = Session(self.name, self.root_url)

         self.session.use_doc('{0} {1}'.format(self.name, datetime.now()))
         self.document = Document()
         self.session.load_document(self.document)

         layout = self.get_bokeh_document()

         self.document.add(layout)

         self.session.store_document(self.document)

         self.bokeh_docid = self.session.docid
         self.bokeh_objid = self.document.context._id

          t = Thread(target=self.run)
          t.start()

 def run(self):
         print 'Polling document'
         return self.session.poll_document(self.document)
```

**Comment by lukebarnard1** [comment]
I'm going to close this because it's probably unnecessary and confusing. The ```object_page``` directive is probably what we should have used. Django is obsolete. I think this could have been avoided a few months back with better documentation on directives like ```object_page``` and the bokeh-server in general. 
We thought interaction was only possible through creating a session to the bokeh-server from another python process and having the callbacks running in that process rather than from the bokeh-server itself.
